pkg/cmd/attestation/verification/tuf.go

```diff
@@ -0,0 +1,55 @@
+package verification
+
+import (
+	"embed"
+	"fmt"
+	"os"
+
+	"github.com/sigstore/sigstore-go/pkg/tuf"
+)
+
+//go:embed embed
+var embeddedRepos embed.FS
```

Comment:
Are there plans to embed more than `tuf-repo.github.com`?

Reply:
No, we can swap over to just embedding the single file instead of using the file system.